src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -8379,12 +8379,7 @@ void WriteBEPSTable(EnergyPlusData &state)
                 if (produceTabular) {
                     if (state.dataResultsFramework->resultsFramework->timeSeriesAndTabularEnabled()) {
                         state.dataResultsFramework->resultsFramework->TabularReportsCollection.addReportTable(
-                            tableBody,
-                            rowHead,
-                            columnHead,
-                            "Annual Building Utility Performance Summary",
-                            "Entire Facility",
-                            "Site and Source Energy");
+                            tableBody, rowHead, columnHead, "AnnualBuildingUtilityPerformanceSummary", "Entire Facility", "Site and Source Energy");
```

Comment:
Some renames

Reply:
I'd like to hear from some others on this.  I don't have a problem with this table name, but I don't know the impact of this change.  Perhaps @jmarrec you could describe the impact, whether it is necessary, and then @mbadams5 @JasonGlazer could you offer a thumbs up/down/neutral on it?